plugin.yaml

```diff
@@ -6,6 +6,14 @@ plugins:
     package_version: '2.6.5'
     source: https://github.com/cloudify-incubator/cloudify-kubernetes-plugin/archive/2.6.5.zip
 
+dsl_definitions:
+
+  client_config: &client_config
+    client_config:
+      type: cloudify.kubernetes.types.ClientConfig
+      description: Your kubernetes client configuration.
+      required: false
+
```

Comment:
I don't think you need to add any new dsl defs.

Reply:
so what should i do instead?